### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vitalii-borzov**

### CODE REVIEW

File: src/arrayMethodJoin.js [hunk 1]
```diff
@@ -4,8 +4,41 @@
...
+      sep = 'null';
+    }
+
+    for (let i = 0; i < this.length; i++) {
+      if (i === this.length - 1) {
+        if (this[i] === undefined) {
+          result = result + '';
+
+          return result;
+        }
```
Comment (TarasHoliuk) [inline src/arrayMethodJoin.js:26]: this if statement will be redundant if you change logic a bit: add separator only if it's not the last element

Comment (TarasHoliuk) [reply src/arrayMethodJoin.js:26]: and in this case, `return` inside the loop will be redundant

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]

File: src/arrayMethodJoin.js (hunk 1 above)
Comment (vitalii-borzov) [reply src/arrayMethodJoin.js:26]: I can't figure out what to do (
Tell me, please.

Comment (tetiana-ivanchenko) [reply src/arrayMethodJoin.js:26]: why do you need this statement, if you add just empty string it's mean that you didn't add anything

Comment (tetiana-ivanchenko) [reply src/arrayMethodJoin.js:26]: you need just two checks:
1. if current element isn't null and isn't undefined, you shoud add it to resulted string
2. if current element is not last, add separator to your resulted string

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**